pkg/cmd/cli/download/delete.go

```diff
@@ -21,6 +22,7 @@ func NewDeleteCommand(f client.Factory, use string) *cobra.Command {
 		Run: func(c *cobra.Command, args []string) {
 			logLevel := logLevelFlag.Parse()
 			logger := logging.DefaultLogger(logLevel, formatFlag.Parse())
+			logger.Debugf("setting log-level to %s", strings.ToUpper(logLevel.String()))
 			logger.Infof("The command, datamgr download delete, is called")
```

Comment:
Let's change "The command, datamgr download delete, is called " to "Datamgr download delete called"